miden-lib/asm/miden/kernels/tx/note.masm

```diff
@@ -6,16 +6,16 @@ use.miden::kernels::tx::memory
 # ERRORS
 # =================================================================================================
 
-# Input note can not have an empty sender, procedure was likely called from the wrong context
+# Trying to access note sender from incorrect context
 const.ERR_NOTE_INVALID_SENDER=0x00020027
 
-# Input note can not have an empty vault, procedure was likely called from the wrong context
+# Trying to access note vault from incorrect context
 const.ERR_NOTE_INVALID_VAULT=0x00020028
```

Comment:
We don't use the term `note vault` any more, it should instead be `note assets`. So, the error constant could be `ERR_NOTE_INVALID_ASSETS` and the comment could be "Trying to access not assets from incorrect context".